pymc3/distributions/distribution.py

```diff
@@ -96,10 +97,11 @@ def _random(*args, **kwargs):
             class_logp = clsdict.get("logp")
             if class_logp:
 
-                @_logp.register(rv_type)
-                def logp(op, var, rvs_to_values, *dist_params, **kwargs):
-                    value_var = rvs_to_values.get(var, var)
-                    return class_logp(value_var, *dist_params, **kwargs)
+                @_logprob.register(rv_type)
+                def logp(op, value_var, *dist_params, **kwargs):
+                    _dist_params = dist_params[3:]
+                    value_var = value_var[0]
```

Comment:
If `value_var` is not the `value_var`, it should not be called `value_var` in the first place.
(On one side of the `=` it should say something else than `value_var`.)